history/titles/00_k_zandalar.txt

```diff
@@ -9,6 +9,7 @@
 	299.6.5 = { holder = tezlekhan } # Tezlekhan[8330]
 	367.6.10 = { holder = aakul } # A'akul[8330]
 	462.11.11 = { holder = 66000 } # Rastakhan[8330]
+	616.12.11 = { holder = 66001 } # Talanji
```

Comment:
Not all Rasta titles are transferred. Is it intended?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132106394-7598468f-39e8-4bbf-9a53-c6255ad91cf7.png)

</details>

Reply:
Not intended. I thought I gave her all of Rastakhan's titles, but I guess not. I will fix it.

Reply:
Fixed in 9e8a6cfd08a0171e80d38727749aeb463f4db0f5.